test/common/tests.js

```diff
@@ -1315,9 +1315,21 @@ module.exports = (sql, driver) => {
       })
     },
 
+    'force ODBC driver verify' (cfg) {
```

Comment:
I'm not a huge fan of adding this config helper to the object that is meant to expose only tests - it feels a bit out of place and confusing that we have one function here that's not actually a test itself. Given it's only used in the test setup once, I'm not sure it needs to be pulled out to a separate function rather than just used inline.

What is the need to verify this anyway? If it's a test assertion it needs to go in a test case (`it` function) and if it's just a test of the config composition that can go in the unit tests where other config tests are.

Reply:
I agree with your assessment, testing what was modified in the function wasn't straightforward at all to me. I couldn't see a way to call _poolCreate() without a valid config that connects to a server because it in turn calls msnodesql.open(...). I thought about extracting the whole thing to its own _parseConnectionString(...) but that seemed a little overkill and I didn't want to mess with your code that much. All I could think to test at that point was to open an ODBC connection. And that's definitely not a unit test.